2022/questions.md

```diff
@@ -0,0 +1,531 @@
+* Used to have one, but it ended
+* Have never heard of an open source program
+* Haven’t considered it
+* Don’t use or participate in open source
+* Organization is too small to need one
+* Organization’s open source use and participation is too small to need one
+* Don’t see the business value
+* Don’t want to regulate or standardize open source practices
+* Time or resource constraints
+* Want one but can’t justify it
+* Other (please specify)
+
+28. Would your company benefit from an open source program?
+* Yes
+* No
+* Unknown
+
+>Routing: "Yes" goes to Part 4b, "No" goes to Part 4c
+## Part 4b: Not Using - Yes, employer would benefit
+29. How would you approach creating an open source program within your organization?
+* {open ended}
+
+30. What are the top three ways your company would benefit from an open source program? (Choose three)
+* Increased developer recruitment and retention
+* Increased speed and agility in development cycle
+* Better license compliance
+* Lower licensing fees
+* Lower support costs
+* More influence in open source communities
+* Increased contributions to in-house open source projects from external or third-party contributors
+* More awareness of open source use and commercial dependencies
+* Increased market adoption of open source projects
+* Increased participation in external open source projects
+* Faster time to market with new products
+* Increased innovation
+* Culture change, with improved interaction among departments
+* Better security testing and vulnerability management
+* Other (please specify)
+>Routing: go to Part 5a
+### Part 4c: Not Using - No, employer would not benefit
+31. Why not?
+* Used to have one, but it ended
+* Have never heard of an open source program
+* Haven’t considered it
+* Don’t use or participate in open source
+* Organization is too small to need one
+* Organization's open source use and participation is too small to need one
+* Don’t see the business value
+* Don’t want to regulate or standardize open source practices
+* I don’t know
+* Other (please specify)
+>Routing: go to Part 5a
+## Part 5
+### Part 5a: Value of Companies and Foundations
+32. "The following list represents a range of large companies that participate in open source communities. To what degree do you perceive each of them to be “good open source community citizens” in terms of contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem?"
+>{Excellent Above Average Average Below Average Very Poor Don't know}
+* AWS
+* Comcast 
+* Facebook
+* Google
+* IBM
+* Microsoft
+* Red Hat
+* SAP
+* Uber
+* Verizon
+* VMware
+33. Excluding the aforementioned companies, please nominate up to three technology or software companies that exemplify good open source community citizenship in terms of contributions, collaboration and leadership on projects and initiatives within the open source ecosystems?
```

Comment:
```suggestion

33. Excluding the aforementioned companies, please nominate up to three technology or software companies that exemplify good open source community citizenship in terms of contributions, collaboration and leadership on projects and initiatives within the open source ecosystems?
```

In really long lists where items have sub-lists (like this survey) once again, a full line between list items helps markdown render correctly <3